src/charm.py

```diff
@@ -172,6 +208,35 @@ def _is_configuration_valid(self) -> bool:
             valid = False
         return valid
 
+    # TODO: Use when relation with Or8cr is ready.
+    @staticmethod
+    def _is_valid_domain(domain: str) -> bool:
```

Comment:
When we start using relation, I don't think we'll need this check, because Orc8r side will validate the domain.

Reply:
Agree, removed.